src/vsstls.cpp

```diff
@@ -461,11 +465,24 @@ void VSStls::doIterations() {
   auto func = [this](double alphaTmp)->double{return alphaDifference(alphaTmp);};
   SecantSolver rsol(in.getErrMinAlpha(), in.getNIterAlpha());
   rsol.solve(func, in.getAlphaGuess());
-  if (!rsol.success()) {
-    throw runtime_error("VSStls: the root solver did not converge to the desired accuracy.");
-  }
+  const double& theta = in.getDegeneracy(); 
```

Comment:
Remove all these lines of dead code. In general do not leave lines of commented code in the source code